templates/kubernetes/data-sources/vkcs_kubernetes_clustertemplates.md.tmpl

```diff
@@ -13,5 +13,5 @@ description: |-
 ## Example Usage
 
 Enabled VKCS Kubernetes Cluster Templates:
-{{tffile .ExampleFile}}
+{{tffile "examples/kubernetes/clustertemplates/main.tf"}}
```

Comment:
why do not .ExampleFile?

Reply:
it will not work for datasources i asked